controls/anssi.yml

```diff
@@ -904,10 +904,11 @@ controls:
     notes: >-
       The rules below create and configure a group named sudogrp, to change the group customize the
       value of var_sudo_dedicated_group.
-    status: automated
+    status: partial
```

Comment:
It would be nice to update the notes informing what need to be done to make it "automated" or why it can't be automated.